packages/lit-element/package.json

```diff
@@ -14,9 +14,25 @@
       "development": "./development/lit-element.js",
       "default": "./lit-element.js"
     },
-    "./lib/": {
-      "development": "./development/lib/",
-      "default": "./lib/"
+    "./css-tag.js": {
```

Comment:
Also need to update the "files" section of this package.json or else they won't get published

Reply:
Updated.